### DISCUSSION TITLE
Voice sounds like strange woman lol

### DISCUSSION THREAD

**Discussion by Sampire23** [voicepaw/so-vits-svc-fork#950] [Q&A]
Hey guys. So I have 3:26 of a rapper friend of mine rapping. I sliced it up into 99 samples, all under 10 seconds. High quality dry studio recordings with no effects. For some reason regardless of if I select pitch prediction or adjust manually, his voice just comes out sounding like some strange woman haha. (he sounds nothing like that)  

my config file looks like this incase it helps offer any answers:

{
  "train": {
    "log_interval": 200,
    "eval_interval": 25000,
    "seed": 1234,
    "epochs": 1668,
    "learning_rate": 0.0001,
    "betas": [
      0.8,
      0.99
    ],
    "eps": 1e-09,
    "batch_size": 16,
    "fp16_run": false,
    "bf16_run": false,
    "lr_decay": 0.999875,
    "segment_size": 10240,
    "init_lr_ratio": 1,
    "warmup_epochs": 0,
    "c_mel": 45,
    "c_kl": 1.0,
    "use_sr": true,
    "max_speclen": 512,
    "port": "8001",
    "keep_ckpts": 3,
    "num_workers": 4,
    "log_version": 0,
    "ckpt_name_by_step": false,
    "accumulate_grad_batches": 1
  },
  "data": {
    "training_files": "filelists/44k/train.txt",
    "validation_files": "filelists/44k/val.txt",
    "max_wav_value": 32768.0,
    "sampling_rate": 44100,
    "filter_length": 2048,
    "hop_length": 512,
    "win_length": 2048,
    "n_mel_channels": 80,
    "mel_fmin": 0.0,
    "mel_fmax": 22050,
    "contentvec_final_proj": false
  },
  "model": {
    "inter_channels": 192,
    "hidden_channels": 192,
    "filter_channels": 768,
    "n_heads": 2,
    "n_layers": 6,
    "kernel_size": 3,
    "p_dropout": 0.1,
    "resblock": "1",
    "resblock_kernel_sizes": [
      3,
      7,
      11
    ],
    "resblock_dilation_sizes": [
      [
        1,
        3,
        5
      ],
      [
        1,
        3,
        5
      ],
      [
        1,
        3,
        5
      ]
    ],
    "upsample_rates": [
      8,
      8,
      2,
      2,
      2
    ],
    "upsample_initial_channel": 512,
    "upsample_kernel_sizes": [
      16,
      16,
      4,
      4,
      4
    ],
    "n_layers_q": 3,
    "use_spectral_norm": false,
    "gin_channels": 256,
    "ssl_dim": 768,
    "n_speakers": 200,
    "type_": "hifi-gan",
    "pretrained": {
      "D_0.pth": "https://huggingface.co/datasets/ms903/sovits4.0-768vec-layer12/resolve/main/sovits_768l12_pre_large_320k/clean_D_320000.pth",
      "G_0.pth": "https://huggingface.co/datasets/ms903/sovits4.0-768vec-layer12/resolve/main/sovits_768l12_pre_large_320k/clean_G_320000.pth"
    }
  },
  "spk": {
    "gdub": 0
  }
}
